src/promptflow/promptflow/_sdk/_service/templates/ui_traces.j2

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Table</title>
+    <style>
+        table, th, td {
+            border: 1px solid black;
+            border-collapse: collapse;
+        }
+        th, td {
+            padding: 5px;
+            text-align: left;
+        }
+    </style>
+</head>
+<body>
+    <table>
+        <thead>
+            <tr>
+                <th>trace_id</th>
```

Comment:
May directly put a json here, so UI can copy for them to demo

Reply:
updated, keep current table, and add plain JSON string below